nixos/modules/services/networking/prosody.nix

```
@@ -744,11 +777,12 @@ in
             You need to setup at least a MUC domain to comply with
             XEP-0423.
           '' + genericErrMsg;}
-        { assertion = cfg.uploadHttp != null || !cfg.xmppComplianceSuite;
+        { assertion = cfg.uploadHttp != null || cfg.httpFileShare != null || !cfg.xmppComplianceSuite;
           message = ''
-            You need to setup the uploadHttp module through
-            config.services.prosody.uploadHttp to comply with
-            XEP-0423.
+            You need to setup the http_upload or http_file_share modules through
+            config.services.prosody.uploadHttp
+            or config.services.prosody.httpFileShare
+            to comply with XEP-0423.
```

```suggestion
            You need to setup the http_upload or http_file_share modules through config.services.prosody.uploadHttp
            or config.services.prosody.httpFileShare to comply with XEP-0423.
```
we don't have a strict line limit, so we might as well format this to be somewhat readable.